src/foam/nanos/pii/test/PIIReportTest.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * NANOPAY CONFIDENTIAL
```

Comment:
Code in the open source repo shouldn't be NANOPAY CONFIDENTIAL

Reply:
Missed on the copy. updated.